exo6.py

```diff
@@ -7,4 +7,5 @@
 xB = input('Entrez xB : ')
 yB = input('Entrez yB : ')
 d = (int(xB)-int(xA))**2+(int(yB)-int(yA))**2
-print ('Distance AB =', d) 
+print ('Distance') 
```

Comment:
il me semble qu'il est plus correcte de faire un print sur une seule ligne 
```suggestion
print ('Distance ') 
```